tests.js

```diff
@@ -575,6 +626,21 @@ window.Specs = {
 			"first-snapshot": 1998,
 			"last-snapshot": 1998
 		},
+		"@rules": {
+			"@page": {
+				"links": {
+					"tr": "#page-box",
+					"dev": "#page-box"
+				},
+				"tests": [
+					"@page { margin: 2cm; }",
+					"@page :left { margin: 2cm; }",
+					"@page :right { margin: 2cm; }",
+					"@page :first { margin: 2cm; }",
+					"@page :blank { margin: 2cm; }"
```

Comment:
I don't see `:blank` included in CSS2. It seems it was introduced in Paged Media 3.

Also, I think you could add dedicated tests for margin and its longhands below, as descriptors.

Reply:
Very good points! I've moved the test for `:blank` back to Paged Media 3 and added the `margin-*` longhand tests here.

Reply:
I mean, we have a dedicated way to test descriptors, like what has been done for `@counter-style example` or `@font-face` for example. Could we use that instead for `margin` and its longhands?